sample-docker-app/docker-compose.yml

```
@@ -1,4 +1,4 @@
 version: '2.0'
 services:
   app:
-    image: sample-docker-app
\ No newline at end of file
+    image: sample-docker-app-x86_64
```

This should stay as it was before
```suggestion
    image: sample-docker-app

```